frontend/packages/console-plugin-sdk/src/codegen/__tests__/plugin-integration-tests.spec.ts

```diff
@@ -0,0 +1,95 @@
+import { PluginPackage, Package, filterActivePluginPackages } from '../plugin-resolver';
+import { getPluginIntegrationTests } from '../plugin-integration-tests';
+import { templatePackage } from './plugin-resolver.spec';
+
+const appPackage: Package = {
+  ...templatePackage,
+  name: 'app',
+  dependencies: {
+    '@console/bar-plugin': '1.2.3',
+    '@console/qux-plugin': '2.3.4',
+  },
+};
+const monorepoRootDir = process.cwd();
+
+describe('getPluginIntegrationTests', () => {
+  it('will not fail on missing tests.', () => {
+    const pluginPackages: PluginPackage[] = [
+      {
+        ...templatePackage,
+        name: '@console/qux-plugin',
+        version: '2.3.4',
+        consolePlugin: { entry: 'plugin.ts' },
+      },
+    ];
+    const pluginFilter = () => filterActivePluginPackages(appPackage, pluginPackages);
+    const expectedTests = {};
+
+    expect(getPluginIntegrationTests(monorepoRootDir, pluginFilter)).toEqual(expectedTests);
+  });
+
+  it('will not pick none active packages.', () => {
```

Comment:
This concern is unrelated to `getPluginIntegrationTestSuites` function, I'd remove this test case.

`getPluginIntegrationTestSuites` should accept the list of plugin packages, it's not important how these are resolved.